src/mmfatl.c

```diff
@@ -28,69 +28,20 @@
 #include "mmfatl.h"
 
 /*!
- * string terminating character
+ * some ASCII control characters
  */
 enum {
   NUL = '\x00',
+  LF = '\n',
 };
 
-//----------
 
-/*
- * During development you may not want to expose preliminary results to the
- * normal compile, as this would trigger 'unused' warnings, for example.  In
- * the regression test environment your code may be referenced by testing code,
- * though.
- *
- * This section should be empty, or even removed, once your development is
- * finished.
- */
-#ifdef TEST_ENABLE
-#   define UNDER_DEVELOPMENT
-#endif
+//***     utility code used in the implementation of the interface    ***
```

Comment:
```suggestion
// ***    utility code used in the implementation of the interface    ***
```
It is at best misleading and at worst causes miscompiles if you have a line comment starter that also looks like a block comment starter.